packaging/anaconda-webui.spec.in

```diff
@@ -11,7 +11,19 @@ BuildRequires:  libappstream-glib
 BuildRequires:  make
 BuildRequires:  gettext
 
-%global anacondacorever 40.20
+%if %{?fedora} < 40
+# Web UI is not supported on Fedora < 40
+%global anacondacorever 1000
+%endif
+
+%if %{?fedora} >= 40
+%global anacondacorever 40.22.2
+%endif
+
+%if %{?fedora} >= 41
+%global anacondacorever 41.2
```

Comment:
Can you explain why these versions like this?  So I assume this PR does not work with version less than `40.22.2`. But why `41.2` already in F41 (which only branched now). I assume that Fedora versions their packages by fedora version? (that must be hella confusing! :D) If so, than 40.22.2 == 41.2? So 41.0 and 41.1 do not work?

Reply:
Anaconda tags the different branches with different tags, which start with the Fedora version number. So branch master, is now targetting Fedora 41, which and 41.2 is the second release, on that branch after the branching of Fedora 40.

Based on Fesco's decision, about Fedora 40 not containing Web UI, I will remove the conditional here, as it's not needed.

@marusak and yes it's a bit confusing coming from cockpit with single number as tag.